client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -13,7 +13,6 @@ import {
 	isEcommerce,
 	isGSuiteOrExtraLicenseOrGoogleWorkspace,
 	isGSuiteOrGoogleWorkspace,
-	isJetpackBusinessPlan,
```

Comment:
@Automattic/shilling This fixes the `'isJetpackBusinessPlan' is defined but never used. (no-unused-vars)` error.

Reply:
Thanks! I wonder how that got past the tests.